staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas_test.go

```diff
@@ -327,12 +327,12 @@ func TestEstimateMaxLengthJSON(t *testing.T) {
 						Type: "object",
```

Comment:
revert changes to this file?

Reply:
reverted, these changes are from the update vendor (to bring in otel pkgs) I _think_